files/build_templates/docker_image_ctl.j2

```diff
@@ -259,6 +259,16 @@ function postStartAction()
             # This flag will be set to "1" after DB migration/initialization is completed as part of config-setup
             $SONIC_DB_CLI CONFIG_DB SET "CONFIG_DB_INITIALIZED" "0"
         else
+            if [[ "$BOOT_TYPE" != "warm" ]]; then
+                $SONIC_DB_CLI CONFIG_DB SET "CONFIG_DB_INITIALIZED" "0"
+            fi
+            # this is not a first time boot to a new image. Datbase container starts w/ old pre-existing config
+            if [[ -x /usr/local/bin/db_migrator.py ]]; then
```

Comment:
I am thinking if we not follow the same flow as we had earlier. Rather than that, why not confine all the migration logic in one place - config-setup.service?

This would ensure that migration also happens after the DB is loaded w/ relevant config - from minigraph, old config, new pushed config, etc. 

Specifically, I am asking that this db_migrator call be moved here as part of new `else` block:
https://github.com/sonic-net/sonic-buildimage/blob/c05af46d02db315b3354a6f88eef45d49ff85b08/files/image_config/config-setup/config-setup#L370

and to https://github.com/sonic-net/sonic-buildimage/blob/c05af46d02db315b3354a6f88eef45d49ff85b08/files/image_config/config-setup/config-setup#L382

Reply:
We have to run it here since the config_db.json has to be loaded first. If we move this to config-setup, this can't be guaranteed.

https://github.com/sonic-net/sonic-buildimage/pull/16116/files#diff-130e5ab75471398db06b73a4c6e4b56517f42e3c1f17870b5309620fc0e2fafbR247